File: DayThree/for_loop_prime_number.py
my_num = int(input("Enter a number: "))

num_of_factor = 0
for divisor in range(1, my_num + 1):
  if my_num % divisor == 0:
    num_of_factor += 1

if num_of_factor == 2:
  print(my_num, "is a prime number")
else:
  print(my_num, "is not a prime number")

File: DayThree/for_loop_prime_number.swift
print("Enter the number:", terminator: " ")
let userInput = readLine()
let numberInput = userInput!
let myNum = Int(numberInput) ?? 0

var numberOfFactor: Int = 0

for divisor in stride(from: 1, to: myNum + 1, by: 1) {
  if myNum % divisor == 0 {
    numberOfFactor += 1
  }
}

if numberOfFactor == 2 {
  print("\(myNum) is a prime number!")
} else {
  print("\(myNum) is not a prime number :D")
}
